src/ResourceManager/Profile/Commands.Profile.Test/AutosaveTests.cs

```diff
@@ -13,6 +13,9 @@
 // ----------------------------------------------------------------------------------
 
 using Microsoft.Azure.Commands.Common.Authentication;
+#if NETSTANDARD
```

Comment:
Ask Yanni about how these ifdefs should be tagged (there is a text string he is using so there is a uniform cpomment for all ifdefs that will be removed after the last azurerm release